### ISSUE TITLE
UV Support

### ISSUE THREAD

**Issue by My1** [shimunn/fido2luks#37]
rather than using a salt as a pseudo password for this, does this also support using UV from the FIDO device basically using the device and its PIN, FP Sensor or whatever to unlock (and storing the salt in the bootloader which is already supported on its own)?

**Comment by shimunn** [comment]
PINs are supported by the fido2luks binary but not by most of the distribution specific scripts. An static salt(or empty) can be used which will then be included into the initramfs.

**Comment by My1** [comment]
yeah that you can store the salt in the initram is something I have seen but obviously when you wanna have 2 factor, something you have to know like the PIN would be cool.

could PIN support be added to the scripts?
